imager/imager/urls.py

```diff
@@ -1,10 +1,28 @@
 from django.conf.urls import patterns, include, url
 from django.contrib import admin
+from imager import views
+from imager import settings
+from django.conf import settings as dcs
+from django.conf.urls.static import static
 
 urlpatterns = patterns('',
     # Examples:
-    # url(r'^$', 'imager.views.home', name='home'),
+    url(r'^$', 'imager.views.home', name='home'),
     # url(r'^blog/', include('blog.urls')),
-
+    url(r'^accounts/', include('registration.backends.default.urls')),
     url(r'^admin/', include(admin.site.urls)),
+    # url(r'^login/$', views.user_login, name='login'),
+    # url(r'^login/$', 'django.contrib.auth.views.login', name='login'),
+    # url(r'^logout/$', 'django.contrib.auth.views.logout',),
+    # url(r'^profile/edit/', include('profiles.urls'), name='profile_edit'),
```

Comment:
Try not to create giant graveyards of dead code in comments. It makes everything harder to read. It's time to move on.